README.md

```diff
@@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
```

Comment:
50 seems overly strict, especially if we want a prefix.  Looking through our 1113 non-merge commits (with `--format=%s`) we only have 62% ≤50 chars, and but 78% are ≤ 70 chars.  Of course some of those are because folks forgot the blank-line after the summary.  But I still think a larger limit here would help encourage more explicit summaries.